frontend/containers/search-auto-suggestion/component.tsx

```diff
@@ -0,0 +1,148 @@
+}) => {
+  const [autoSuggestions, setAutoSuggestions] = useState<Enum[]>();
+  const { data, isLoading } = useEnums();
+  const { page, search, sorting, ...initialFilters } = useQueryParams();
+  const { push } = useRouter();
+  const filterNames = useFilterNames();
+
+  const filters: Enum[] = useMemo(() => {
+    if (!isLoading) {
+      const { category, ticket_size, instrument_type, impact, sdg } = data;
+      return [category, ticket_size, instrument_type, impact, sdg].flat();
+    }
+  }, [isLoading]);
+
+  const closeSuggestions = useCallback(() => {
+    onChangeOpenSuggestion(false);
+    setAutoSuggestions(undefined);
+  }, [onChangeOpenSuggestion]);
+
+  useEffect(() => {
+    if (searchText.length > 1) {
+      const filtersToSuggest = filters
+        ?.filter(({ name }) => name.toLowerCase().includes(searchText.toLowerCase()))
+        .sort((a) => {
+          // Return the ones that match the beginning of the search text first
+          return a.name.toLowerCase().startsWith(searchText.toLowerCase()) ||
+            a.id[0] === searchText.toLowerCase()[0]
+            ? -1
+            : 1;
+        });
+      if (filtersToSuggest?.length) {
+        onChangeOpenSuggestion(true);
+        setAutoSuggestions(filtersToSuggest);
+        return;
+      }
+      closeSuggestions();
+    }
+    closeSuggestions();
+  }, [closeSuggestions, filters, onChangeOpenSuggestion, searchText]);
+
+  const handleFilter = (filter: Enum) => {
+    const filterKey = `filter[${filter.type}]`;
+    const newFilters = omit(initialFilters, filterKey);
+    push(
+      {
+        query: {
+          page: 1,
+          search,
+          sorting,
+          ...newFilters,
+          [filterKey]: filter.id,
+        },
+      },
+      undefined,
+      {
+        shallow: true,
+      }
+    );
+    closeSuggestions();
+  };
+
+  const handleSearch = () => {
+    push(
+      {
+        query: {
+          page: 1,
+          search: searchText,
+          sorting,
+          ...initialFilters,
+        },
+      },
+      undefined,
+      {
+        shallow: true,
+      }
+    );
+    closeSuggestions();
+  };
+
+  return (
+    <div
+      id="filters"
+      role="region"
+      aria-labelledby="filters-button"
+      className={cx(
+        'h-0 w-full bg-white -mt-1 rounded-b-4xl drop-shadow-xl transition-all ease-in',
+        {
+          'h-fit border-t-gray-200 border-t-2 overflow-hidden': !!autoSuggestions,
+        }
+      )}
+    >
+      {!!autoSuggestions?.length && (
+        <div className="px-9 py-5">
+          <div>
+            <Button key="custom" theme="naked" className="px-0 py-2" onClick={handleSearch}>
+              &quot;{searchText}&quot;
+              <span className="text-gray-600 ml-2">
+                <FormattedMessage defaultMessage="Custom search" id="AoHRBG" />
+              </span>
+            </Button>
+          </div>
+          <div>
+            <p className="text-sm text-gray-800">
```

Comment:
```suggestion
            <p className="py-2 text-sm text-gray-800">
```

Reply:
<img width="792" alt="Screenshot 2022-06-13 at 12 47 53" src="https://user-images.githubusercontent.com/6273795/173347154-5e48c020-8b82-4a3d-8538-3e7d7d01c191.png">

Just to give some more spacing between this label and the items.